github/billing.go

```diff
@@ -18,30 +18,49 @@ type BillingService service
 
 // ActionBilling represents a GitHub Action billing.
 type ActionBilling struct {
-	TotalMinutesUsed     int                  `json:"total_minutes_used"`
-	TotalPaidMinutesUsed int                  `json:"total_paid_minutes_used"`
-	IncludedMinutes      int                  `json:"included_minutes"`
-	MinutesUsedBreakdown MinutesUsedBreakdown `json:"minutes_used_breakdown"`
+	TotalMinutesUsed     int                  `json:"total_minutes_used,omitempty"`
+	TotalPaidMinutesUsed int                  `json:"total_paid_minutes_used,omitempty"`
+	IncludedMinutes      int                  `json:"included_minutes,omitempty"`
+	MinutesUsedBreakdown MinutesUsedBreakdown `json:"minutes_used_breakdown,omitempty"`
 }
 
 type MinutesUsedBreakdown struct {
-	Ubuntu  int `json:"UBUNTU"`
-	MacOS   int `json:"MACOS"`
-	Windows int `json:"WINDOWS"`
+	Ubuntu  int `json:"UBUNTU,omitempty"`
+	MacOS   int `json:"MACOS,omitempty"`
+	Windows int `json:"WINDOWS,omitempty"`
 }
 
 // PackageBilling represents a GitHub Package billing.
 type PackageBilling struct {
-	TotalGigabytesBandwidthUsed     int `json:"total_gigabytes_bandwidth_used"`
-	TotalPaidGigabytesBandwidthUsed int `json:"total_paid_gigabytes_bandwidth_used"`
-	IncludedGigabytesBandwidth      int `json:"included_gigabytes_bandwidth"`
+	TotalGigabytesBandwidthUsed     int `json:"total_gigabytes_bandwidth_used,omitempty"`
+	TotalPaidGigabytesBandwidthUsed int `json:"total_paid_gigabytes_bandwidth_used,omitempty"`
+	IncludedGigabytesBandwidth      int `json:"included_gigabytes_bandwidth,omitempty"`
 }
 
 // StorageBilling represents a GitHub Storage billing.
 type StorageBilling struct {
-	DaysLeftInBillingCycle       int     `json:"days_left_in_billing_cycle"`
-	EstimatedPaidStorageForMonth float64 `json:"estimated_paid_storage_for_month"`
-	EstimatedStorageForMonth     int     `json:"estimated_storage_for_month"`
+	DaysLeftInBillingCycle       int     `json:"days_left_in_billing_cycle,omitempty"`
+	EstimatedPaidStorageForMonth float64 `json:"estimated_paid_storage_for_month,omitempty"`
+	EstimatedStorageForMonth     int     `json:"estimated_storage_for_month,omitempty"`
+}
+
+// ActiveCommitters represents the total active committers across all repositories in an Organization.
+type ActiveCommitters struct {
+	TotalAdvancedSecurityCommitters int                          `json:"total_advanced_security_committers,omitempty"`
```

Comment:
If it isn't a reference type (e.g. `*int` in this case) then it doesn't need the `omitempty`.

```suggestion
	TotalAdvancedSecurityCommitters int                          `json:"total_advanced_security_committers"`
```

Reply:
@gmlewis can I know how we find out if its a reference type or not?

Reply:
> @gmlewis can I know how we find out if its a reference type or not?

There are a few scenarios where reference types are needed and/or desired:

* When the response might not populate the field,
* When an option being sent from the client to the server is optional and we don't want the `zero` value of the field to be sent
* When we have a slice of a struct, it is desirable to iterate over a slice of pointers rather than a slice of values.

Conversely, if a field is already a reference type (for example, a slice or an interface), we don't typically want to have a pointer to it.  In other words, if you ever see `*[]something` or *[]*something` then that is typically suspect and not desirable. 

I hope that helps.